### PR TITLE
[15.0] [mig] l10n_pt_account_invoicexpress 

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**
Dependent on PR ( https://github.com/OCA/l10n-portugal/pull/74 )

### CODE REVIEW

**Comment by dreispt** [comment]
Thank you!

File: l10n_pt_account_invoicexpress/data/mail_template.xml [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+            <field name="name">InvoiceXpress: Send Invoice by Email</field>
+            <field name="model_id" ref="account.model_account_move"/>
+            <field name="subject">Sua Fatura ${object.name | safe}</field>
+            <field name="email_to">${object.partner_id.email | safe}</field>
+            <field name="email_cc">${object.env.user.email | safe}</field>
+            <field name="body_html" type="html">
+                <p>
+                    Olá,
+                    <br/>
+                    Enviamos em anexo a fatura ${object.name | safe}
```
Comment (dreispt) [inline l10n_pt_account_invoicexpress/data/mail_template.xml]: With v15 email templates needs to converted to QWeb syntax.

**Review by dreispt — CHANGES_REQUESTED** [review]

**Comment by dreispt** [comment]
Something wring found in the tests. Could be a pre-existing bug surfaced now:
```
ValueError: Wrong @depends on '_compute_invoicexpress_doc_type' (compute method of field account.move.invoicexpress_doc_type). Dependency field 'partner_shipping_id' not found in model account.move.
```

**Comment by rafaelbn** [comment]
Hello @heliaktiv can you please review comments from @dreispt ? Thank you!! 🙏 😄

**Pull request closed without merging**